Fix additional tutorials (#388)
cylc/cylc-doc: src/tutorial/furthertopics/clock-triggered-tasks.rst

Note the different values used for the cycle offsets of the clock-trigger tasks.

Save your changes, install and run your workflow using::

   cylc validate .
   cylc install
   cylc play clock-trigger

.. note::

   Setting ``initial cycle point = now`` will run your workflow using the
   current time at startup as the initial cycle point.

Again, notice how the tasks trigger until the current time is reached.

> Leave your workflow running for a while to confirm it is working as expected
> and then shut it down using the :guilabel:`stop` button in the
> :ref:`tutorial.gui`.

Reviewer: No mention of the GUI in this tutorial hitherto
```suggestion
Leave your workflow running for a while to confirm it is working as expected
before stopping it.
```
